kornia/core/check.py

```diff
@@ -392,23 +392,18 @@ def KORNIA_CHECK_IS_IMAGE(x: Tensor, msg: Optional[str] = None, raises: bool = T
         True
 
     """
-    res = KORNIA_CHECK_IS_COLOR_OR_GRAY(x, msg, raises=raises)
-
-    if not raises and not res:
-        return False
-
-    err_msg = f"Invalid image value range. Expect [0, 1] but got [{x.min()}, {x.max()}]."
-    if msg is not None:
-        err_msg += f"\n{msg}"
-
-    if x.dtype in [float16, float32, float64] and (x.min() < 0.0 or x.max() > 1.0):
-        if raises:
-            raise ValueError(err_msg)
-        return False
-    elif x.min() < 0 or x.max() > 2**bits - 1:
-        if raises:
-            raise ValueError(err_msg)
-        return False
+    # Combine the color or gray check with the range check
+    if not KORNIA_CHECK_IS_COLOR_OR_GRAY(x, msg, raises):
+        return False if not raises else None
```

Comment:
which is the difference between returning False or None? i don't get why we need return None

Reply:
you're right. making the fix